src/set_theory/game/pgame.lean

```diff
@@ -148,6 +148,11 @@ def move_right : Π (g : pgame), right_moves g → pgame
 @[simp] lemma right_moves_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).right_moves = xr := rfl
 @[simp] lemma move_right_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).move_right = xR := rfl
 
+/-- Conway induction on games. -/
```

Comment:
I'd argue this is no more "Conway induction" than `rec_on` is; although I guess `rec_on` has no doc entry.

So maybe something like
```suggestion
/-- A variant of `pgame.rec_on` expressed in terms of `pgame.move_left` and `pgame.move_right`.

Both this and `pgame.rec_on` describe Conway induction on games. -/
```